src/components/PageNavigation/PageNavigation.tsx

```diff
@@ -94,27 +98,39 @@ export const PageNavigation: FC<AppNavigationProps> = ({ logo }) => {
                 <Navigation>
                     <NavigationSection title={tr('Goals')}>
                         {goalsRoutes.map(({ title, href }) => (
-                            <NavigationItem key={href} selected={!isPresetActive && activeRoute === href} href={href}>
-                                {title}
-                            </NavigationItem>
+                            <div key={href} className={s.PageNavigationItemLink}>
+                                <NextLink href={href} legacyBehavior>
+                                    <NavigationItem selected={!isPresetActive && activeRoute === href} href={href}>
```

Comment:
NavigationItem with link is repeated, it can be moved to a small utility component in the same file